etc/config/cuda.amazon.properties

```diff
@@ -88,3 +88,12 @@ libs.cudacxx.versions=trunk
 libs.cudacxx.url=https://github.com/nvidia/libcudacxx
 libs.cudacxx.versions.trunk.version=trunk
 libs.cudacxx.versions.trunk.path=/opt/compiler-explorer/libs/libcudacxx/trunk/include
+
+libs.nvtx.name=NVIDIA Tools Extension
+libs.nvtx.description=APIs for code annotation 
+libs.nvtx.versions=dev:301
+libs.nvtx.url=https://github.com/NVIDIA/NVTX
+libs.nvtx.versions.dev.version=dev
+libs.nvtx.versions.dev.path=/opt/compiler-explorer/libs/nvtx/dev/cpp:/opt/compiler-explorer/libs/nvtx/dev/c/include
+libs.nvtx.versions.301.version=3.0.1
+libs.nvtx.versions.301.path=/opt/compiler-explorer/libs/nvtx/3.0.1/cpp:/opt/compiler-explorer/libs/nvtx/3.0.1/c/include
```

Comment:
I'm going to merge then change this to .../v3.0.1/... as that's what our installer ended up doing...